tests/tests.bats

```diff
@@ -236,6 +299,20 @@ teardown() {
   [[ "$output" == *"SUCCESS"* ]]
 }
 
+@test "ping-serve spt" {
+  UNIKERNEL=test_ping_serve/test_ping_serve.spt
+
+  [ $(id -u) -ne 0 ] && skip "Need root to run this test, for ping -f"
+
+  (
+    sleep 1
+    ${TIMEOUT} 30s ping -fq -c 100000 ${NET_IP} 
+  ) &
+
+  run ${TIMEOUT} --foreground 30s ${SPT_TENDER} --net=${NET} -- ${UNIKERNEL} limit
+  [ "$status" -eq 0 ]
+}
+
 @test "abort hvt" {
```

Comment:
It seems like the `@test "abort spt"` case is missing in this file (`tests/tests.bats`), apart from that the test suite runs for me and crashes in `_exception`,`_zeropage` and `_seccomp` as expected.

Reply:
That's intentional. The "abort" test case is misnamed, and actually hvt-specific as it tests the functionality of the "dumpcore" module. I have it on my list to rename it and also remove the virtio invocation.